app/Http/Controllers/LoanController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+use Illuminate\Foundation\Validation\ValidatesRequests;
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Http\Request;
+use App\Services\LoanService;
+
+class LoanController extends Controller
+{
+    protected $loanService;
+
+    public function __construct(LoanService $loanService)
+    {
+        $this->loanService = $loanService;
+    }
+
+    public function index()
+    {
+        return view('loan.store');
```

Comment:
```suggestion
return view('loan.form');
```
mabye ??

Reply:
i named it form intially but renamed to store just to have transperancy

Reply:
but , it is not working :(